### DISCUSSION TITLE
Design loading - 403 Forbidden

### DISCUSSION THREAD

**Discussion by lukasrst** [Budibase/budibase#7696] [Help]
Hi,
I just installed Budibase via Docker on a Linux Server and the Design Page is loading endless.
The browser developer tools show the error "403 Forbidden".

This error happens with completely fresh apps as well as with templates. Tested Chrome, Chromium, Firefox and Edge with allowed Cookie Setting.

![image](https://user-images.githubusercontent.com/58342398/189082908-ec47d346-d7d3-4ef2-82df-2c928d7b92ab.png)
![image](https://user-images.githubusercontent.com/58342398/189083895-3116c65f-0eb7-4860-a60d-8b8a84320955.png)

Thanks for your help!

**Comment by lukasrst** [comment]
Should be a Bug in v1.3.11 -> Its workin in v1.2.52

**Reply by oitkenmen** [reply]
I agree, v1.2.52 was working fine. Do you know how to downgrade or install a previous version?

**Reply by lukasrst** [reply]
If you are using Docker just install the v.1.2.52 with the tag budibase/budibase:v1.2.52
https://hub.docker.com/r/budibase/budibase/tags

**Reply by cstasius** [reply]
Same issue with 1.3.11 and the docker installation

**Reply by lukasrst** [reply]
Fixed with v1.3.15
